plugins/leemons-plugin-academic-calendar/frontend/src/pages/private/regional/index.js

```diff
@@ -166,13 +169,20 @@ export default function RegionalCalendars() {
 
   function save() {
     form.handleSubmit(async (data) => {
+      const dataToSave = {
+        id: data.id,
+        name: data.name,
+        regionalEvents: data.regionalEvents,
+        localEvents: data.localEvents,
+        daysOffEvents: data.daysOffEvents,
+        center: store.center.id,
+      };
       try {
         store.saving = true;
         render();
 
         await saveRegionalConfig({
-          ...data,
-          center: store.center.id,
+          ...dataToSave,
         });
```

Comment:
If you are creating a new object to save, then you don't need to use object destructuring, as the object mutation won't affect you

Reply:
I create a new hook called "useSaveRegionalConfig" and pass "dataToSave" directly. No destructuring object.